packages/manager/apps/container/src/container/nav-reshuffle/sidebar/SubTree.tsx

```diff
@@ -195,8 +197,8 @@ const SubTree = ({
               </li>
             )}
             {(rootNode.id !== 'pci' || selectedPciProject !== null) &&
-              rootNode.children?.map((node) => (
-                <li key={node.id} id={node.id}>
+              rootNode.children?.map((node, index) => (
+                <li key={node.id} id={node.id} className={index === rootNode.children?.length - 1 ? 'pb-4' : ''}>
```

Comment:
Wouldn't it be easier to handle this with pure css ?
I feel like this is equivalent to:
`
li.pciEntry:last-of-type {
  padding-bottom: 1.5rem !important;
}
`
(it probably need you to add a class on each li)

Reply:
Done